examples/v16/central_system.py

```diff
@@ -31,10 +31,22 @@ def on_boot_notitication(self, charge_point_vendor: str, charge_point_model: str
 
 
 async def on_connect(websocket, path):
-    """ For every new charge point that connects, create a ChargePoint instance
-    and start listening for messages.
-
+    """ For every new charge point that connects, create a ChargePoint
+    instance and start listening for messages.
     """
+    requested_protocols = websocket.request_headers['Sec-WebSocket-Protocol']
```

Comment:
The bounds of `websocket.request_headers` aren't checked here. A client that doesn't send the header 'Sec-WebSocket-Protocol' will cause a `KeyError` here.

Reply:
Agreed 👍